less/variables.less

```diff
@@ -18,7 +18,7 @@
 @redSidebarColor: #BB3C33;
 @pinkSidebarColor: #D64262;
 @footerBackgroundColor: #f1f1f1;
-@mozFestPromoRed: #E12526;
+@thimblePromoGreen: #279761;
```

Comment:
Wondering if we should just give this a more general name instead (i.e., `promoBannerBackground`?) This way we don't have to update the variable name in both `less/variables.less` and `less/pages/home.less` every time we refresh the banner